src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2Projections.cs

```diff
@@ -126,6 +126,17 @@ await context
                         }.ToBytes(_encoding)
                     }, ct);
             });
+
+            When<Envelope<ParcelWasRetiredV2>>(async (context, message, ct) =>
+            {
+                await context.FindAndUpdateParcelExtract(
+                    message.Message.ParcelId,
+                    parcel =>
+                    {
+                        UpdateVersie(parcel, message.Message.Provenance.Timestamp);
```

Comment:
You're not updating the status? Add:
`UpdateRecord(parcel, x => x.status.Value = MapStatus(ParcelStatus.Retired));`